### PR TITLE
refactor(nuxt): update to new design

### LINKED ISSUE #2838: Update Nuxt Icons for Nuxt V3/Nuxt Rebranding
- [x] I'm sure this issue is not a _duplicate_.

New Logo at: https://nuxtjs.org/design/

### PR DESCRIPTION
**Pull request by sasial-dev**
🎉  #3000 🎉 

_**Fixes #2838**_

**Changes proposed:**

- Add new extensions from nuxt 3
- Update to new nuxt icon

### CODE REVIEW

**Comment by sasial-dev** [comment]
@robertohuertasm, I'd like some help with the folder icons: while I've done them, they don't seem to line up right!

**Review by robertohuertasm — COMMENTED** [review]
The folder icons are not well adjusted. No worries. I'll take care of it. Thanks for this PR!

File: src/iconsManifest/supportedExtensions.ts [hunk 1]
```diff
@@ -2898,8 +2898,8 @@ export const extensions: IFileCollection = {
     {
       icon: 'nuxt',
       extensions: [],
-      filenamesGlob: ['nuxt.config'],
-      extensionsGlob: ['js', 'ts'],
+      filenamesGlob: ['nuxt.config', '.nuxtrc', '.nuxtignore'],
```
Comment (robertohuertasm) [inline src/iconsManifest/supportedExtensions.ts:2901]: Regarding both `.nuxtrc` and `.nuxtignore` files, it seems that they should work without extension: https://nuxtjs.org/docs/configuration-glossary/configuration-ignore/

Do  you have any other link where it states that they should be used with extension?

**Review by robertohuertasm — CHANGES_REQUESTED** [review]

File: src/iconsManifest/supportedExtensions.ts (hunk 1 above)
Comment (sasial-dev) [reply src/iconsManifest/supportedExtensions.ts:2901]: yeah they should work without extension, not sure what I was thinking there.

**Comment by sasial-dev** [comment]
I've gone ahead and fixed the icons, this is ready for merge!

**Review by robertohuertasm — APPROVED** [review]
LGTM 🚀 Thanks for your contribution ❤️

**Pull request merged**